style.css

```diff
@@ -340,202 +455,338 @@ i {
   height: 1em;
 }
 
+/* =======TABLET MEDIAQUERIES ======== */
 @media screen and (min-width: 768px) {
   nav a {
     font-size: 14px;
   }
-}
 
-.top-nav {
-  display: flex;
-  background-color: #000;
-  flex-direction: row;
-  justify-content: space-between;
-  padding-bottom: 5px;
-}
+  .top-nav {
+    display: flex;
+    background-color: #000;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-bottom: 5px;
+  }
 
-.mid,
-.btm {
-  display: flex;
-}
+  .mid,
+  .btm {
+    display: flex;
+  }
 
-nav .fa-bars,
-.logo-img {
-  display: none;
-}
+  nav .fa-bars,
+  .logo-img {
+    display: none;
+  }
 
-nav .fa-search {
-  font-size: 13px;
-  padding: 10px;
-}
+  nav .fa-search {
+    font-size: 13px;
+    padding: 10px;
+  }
 
-.btm {
-  align-items: center;
-}
+  .btm {
+    align-items: center;
+  }
 
-/* stylelint-disable */
-.btm-nav a {
-  margin: 0;
-}
-/* stylelint-enable */
+  /* stylelint-disable */
+  .btm-nav a {
+    margin: 0;
+  }
+  /* stylelint-enable */
```

Comment:
- [ ] You disabled stylelint here. Please remove it and solve the linters. 

**It is a bad practice to disable the linters, try to solve it. If it is something that you can't even find on the internet on how to solve it, ask the full-time students chat in the Slack group. If no one else can solve it, then there you resort to disabling the linters.**